### ISSUE TITLE
www.google.com does not invert

### ISSUE THREAD

**Issue by regraded** [darkreader/darkreader#781]
Unable to get www.google.com, or a search results page, to invert.

Is this a known issue? I couldn't find a topic on GitHub. I have tried all of the invert options but none seem to do the trick. 

Edit: System info
Using Chrome 69.0.3497.100 on OSX 10.13.5

**Comment by rolagg** [comment]
All the invert options works for me on this site. 
You need to specify what software (OS, browser, other extensions/addons) you also use to make it possible to help you.

**Comment by regraded** [comment]
My apologies. Added system info above. 

Its odd, I opened up an incognito window so that I could start selectively enabling extensions to see if one of them was causing the issue, I already have dark-reader enabled for incognito, and so I loaded a search page which to my surprise **was** inverted! 

Though, in the background in a non-incognito window, the same page was _not inverted_. Using the toggle button in the incognito window switched the invert on both windows (ingoc window went white, non-incog went dark). 

Now however, both seem to be 'in-sync' and the toggle button in normal Chrome windows is working on google.com pages. 

Maybe my preference from a previous incognito session was overriding the normal preferences? either way, it seems to have resolved itself. yay!
